spec/features/user_signs_out_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'user signs out', %{
+  As an authenticated user
+  I want to sign out
+  So that no one else can post items or reviews on my behalf
+} do
+  # Acceptance Criteria:
+  # [] If I am an authenticated user, I can click a button to sign out.
+  # [] If I am signed out, I can't sign out again.
+
+  before :each do
+    user = FactoryGirl.create(:user)
+    visit new_user_session_path
```

Comment:
if you find yourself using this sign-in logic a lot, consider extracting it to a spec helper!